astroid/brain/brain_regex.py

```diff
@@ -0,0 +1,88 @@
+# Licensed under the LGPL: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
+# For details: https://github.com/PyCQA/astroid/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/astroid/blob/main/CONTRIBUTORS.txt
+
+from __future__ import annotations
+
+from astroid import context, inference_tip, nodes
+from astroid.brain.helpers import register_module_extender
+from astroid.builder import _extract_single_node, parse
+from astroid.const import PY39_PLUS
+from astroid.manager import AstroidManager
+
+
+def _regex_transform() -> nodes.Module:
+    # The RegexFlag enum exposes all its entries by updating globals().
+    # We hard-code the flags for now.
```

Comment:
```suggestion
    """The RegexFlag enum exposes all its entries by updating globals().
    
    We hard-code the flags for now.
    See https://github.com/mrabarnett/mrab-regex/blob/cc392629b28ed0aeb8bf6d46f826ff9e87aac551/regex_3/regex.py#L198
    """
```
To be honest pylint is not the only one having a hard time understanding what the regex code does 😅